### PR TITLE
Content and images updated

### PR DESCRIPTION
**Pull request by stevenixon1999**

### CODE REVIEW

File: privacy-policy/index.html [hunk 1]
```diff
@@ -151,8 +151,8 @@ <h4> [Phone Number]</h4>
       <div class="right-wrap">
         <ul>
           <li><a href="/privacy-policy/">Privacy Policy</a></li>
-          <li><a href="mailto:example@example.com">Email : example@example.com</a></li>
-          <li><a href="tel:+1234567890">Phone: +1 (234) 567-890</a></li>
+          <li><a href="mailto:example@example.com">Email : alphytoursandtravels@gmail.com</a></li>
```
Comment (vaishak101) [inline privacy-policy/index.html]: Check Href value mail to - original email address

File: privacy-policy/index.html [hunk 2]
```diff
@@ -151,8 +151,8 @@ <h4> [Phone Number]</h4>
...
+          <li><a href="tel:+1234567890">Phone:+91 9819274249</a></li>
```
Comment (vaishak101) [inline privacy-policy/index.html]: Check Href value tel - original phone number

File: index.html [hunk 3]
```diff
@@ -475,8 +473,8 @@ <h3 class="title">About Alphy Tours and Travels</h3>
       <div class="right-wrap">
         <ul>
           <li><a href="/privacy-policy/index.html">Privacy Policy</a></li>
-          <li><a href="mailto:example@example.com">Email : example@example.com</a></li>
-          <li><a href="tel:+1234567890">Phone: +1 (234) 567-890</a></li>
+          <li><a href="mailto:example@example.com">Email : alphytoursandtravels@gmail.com</a></li>
```
Comment (vaishak101) [inline index.html]: Href value check

File: index.html [hunk 4]
```diff
@@ -475,8 +473,8 @@ <h3 class="title">About Alphy Tours and Travels</h3>
...
+          <li><a href="tel:+1234567890">Phone: +91 9819274249</a></li>
```
Comment (vaishak101) [inline index.html]: Href value check for phone too

File: index.html (hunk 3 above)
Comment (stevenixon1999) [reply index.html]: Changes Done

**Pull request merged**